docs/MAINTAINERS.adoc

```diff
@@ -190,10 +207,18 @@ LTS Backporting, if needed, will be handled separately by the release team.
 
 Right now we do not have a strict policy about it.
 
+== LTS Process
+
+Jenkins also offers the link:https://jenkins.io/download/lts/[LTS Release Line].
+It is maintained by the Jenkins Release Team which coordinates link:https://jenkins.io/download/lts/#backporting-process[backporting] and release candidate testing.
+It mostly happens through the developer mailing list and through backporting pull requests labeled with link:https://github.com/jenkinsci/jenkins/labels/into-lts[into-lts].
```

Comment:
Ambiguous or wrong: These PRs go directly into LTS, they _are_ the backports.

Reply:
```suggestion
It mostly happens through the developer mailing list. Backports are submitted as pull requests labeled with link:https://github.com/jenkinsci/jenkins/labels/into-lts[into-lts].
```